queries/dune_v2/period_slippage.sql

```diff
@@ -7,27 +8,25 @@ filtered_trades as (
             -- Estimation made here: https://dune.com/queries/1646084
                 then ((gas_used - 73688 - (70528 * num_trades)) / 90000)::int
                 else dex_swaps
-            end as dex_swaps,
+           end as dex_swaps,
            num_trades,
-           solver_address,
-           trader                                              as trader_in,
-           -- Null Receiver: https://dune.com/queries/1729130
-           -- TODO - This could also be fixed in the spellbook!
-           case
-                when receiver = '0x0000000000000000000000000000000000000000'
-                then trader
-            else receiver end                                  as trader_out,
-           sell_token_address                                  as sell_token,
-           buy_token_address                                   as buy_token,
-           atoms_sold,
+           b.solver_address,
+           trader                                       as trader_in,
+           receiver                                     as trader_out,
+           sell_token_address                           as sell_token,
+           buy_token_address                            as buy_token,
+           atoms_sold - coalesce(surplus_fee, 0)        as atoms_sold,
```

Comment:
I wonder if it would make sense to rename the column after the fee is considered to make it more explicit that we are already accounting for surplus (I remember a Uniswap data analysis where they accounted for the LP twice because they didn't realize it was already accounted for in some earlier step of the process).

Something like `atoms_sold - coalesce(surplus_fee, 0) as atoms_sold_after_fee,`

Reply:
That seems reasonable!